buildSrc/src/main/java/Config.kt

```diff
@@ -56,7 +56,7 @@ object Config {
         val androidxCore = "androidx.core:core:1.3.2"
 
         val slf4jApi = "org.slf4j:slf4j-api:1.7.30"
-        val logbackVersion = "1.2.3"
+        val logbackVersion = "1.2.8"
```

Comment:
```suggestion
        val logbackVersion = "1.2.9"
```

also please add an entry to the changelog